src/_content/posts/announcing-lineup.md

```diff
@@ -0,0 +1,57 @@
+---
+author: DjangoCon US Communications Team
+category: General
+published_datetime: 2025-07-03 00:00:00
```

Comment:
I don't know if the published date time is correct? Didn't we say hold on publishing/talking about acceptance until later this month? (July 25?)

Reply:
@emullaney Please share the publish date. There are a few more changes coming in, and I’ll include those as well.

Reply:
Set it to Monday July 14, please.